src/components/HomePage/SortDropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import {
+  ButtonDropdown, DropdownItem, DropdownMenu, DropdownToggle,
+} from 'reactstrap';
+import { SortName } from '../../model/enums/SortName';
+
+export const SortDropdown: FC = () => {
+  const defaultSortName = 'Sort';
+  const [dropdownOpen, setOpen] = React.useState(false);
+  const [SortNameValue, setSortName] = React.useState(defaultSortName);
+
+  const toggle = () => setOpen(!dropdownOpen);
+
+  const sortOptions = [SortName.HIGHEST_RATING, SortName.NEWEST];
+
+  const items = [];
+  for (let i = 0; i < sortOptions.length; i++) {
```

Comment:
I suggest using `sortOptions.map((sortOption) => {...})` instead `for` with `pushes`

Reply:
Fixed in commit 10558d1